source/types/Case.ts

```diff
@@ -1,20 +1,21 @@
 import { EncryptionDetails } from "./Encryption";
 
-export type ApplicationStatusType =
-  | "active:ongoing"
-  | "active:processing"
-  | "active:signature:completed"
-  | "active:signature:pending"
-  | "active:submitted"
-  | "closed"
-  | "notStarted"
-  | "active:submitted:viva"
-  | "active:completionRequired:viva"
-  | "closed:approved:viva"
-  | "closed:completionRejected:viva"
-  | "closed:partiallyApproved:viva"
-  | "closed:rejected:viva"
-  | "notStarted:viva";
+export enum ApplicationStatusType {
+  ActiveOngoing = "active:ongoing",
+  ActiveProcessing = "active:processing",
+  ActiveSignatureCompleted = "active:signature:completed",
+  ActiveSignaturePending = "active:signature:pending",
+  ActiveSubmitted = "active:submitted",
+  Closed = "closed",
+  NotStarted = "notStarted",
+  ActiveSubmittedViva = "active:submitted:viva",
+  ActiveCompletionRequiredViva = "active:completionRequired:viva",
+  ClosedApprovedViva = "closed:approved:viva",
+  ClosedCompletionRejectedViva = "closed:completionRejected:viva",
+  ClosedPartiallyApprovedViva = "closed:partiallyApproved:viva",
+  ClosedRejectedViva = "closed:rejected:viva",
+  NotStartedViva = "notStarted:viva",
```

Comment:
This status does not exists in the api code base

Reply:
Sure thing - I just copied these from the API `caseStatuses.js`.

Reply:
It is okay. :)

Reply:
I double-checked and `notStarted:viva` seems to be used by `createVivaCase` in the API?

Reply:
You're right. My bad. `createVivaCase` will eventually be rewritten to conform to the logic for when workflowid is missing.